news_app.js

```diff
@@ -0,0 +1,105 @@
+// const { doc } = require("prettier");
+
+// FETCH VARIABLES
+const topic = 'food';
+const source = `https://newsapi.org/v2/everything?q=${topic}&sortBy=popularity&pageSize=5&apiKey=b10b22f57fca436ca78791aa2c90a376`;
+
+// DOM VARIABLES
+const body = document.body;
+const news = document.querySelector('.news')
+
+
+// JS FETCH DATA
+const getData = async () => {
+    const response = await fetch(source);
+    if(response.status !== 200){
+        throw new Error('can not fetch the data');
+    };
+    const data = await response.json();
+    return data;
+};
+
+
+// RUNTIME
+getData()
+    .then(data => {
+        // console.log(data);
+        
+        // ! MULTIPLE ARTICLES LOOP
+        // TODO: FIX DATE FORMATTING
+        // TODO: CHECK FOR AUTHOR
+        const newsArticles = data.articles;
+
+        newsArticles.forEach(article => {
+            const {title, author, published, description, source, url, urlToImage} = article;
+            // NEW NEWS DIV
+            const newsDiv = document.createElement('div');
+            // ADD IMAGE
+            const newsPic = document.createElement('img');
+            newsPic.setAttribute('width','400px')
+            newsPic.src = urlToImage
+            newsDiv.appendChild(newsPic);
```

Comment:
consider to wrap pieces of code that are related in a function or even in js module. Then you can write some unit tests for them